opennmt/utils/decoding.py

```diff
@@ -112,6 +112,28 @@ def from_params(params):
         else:
             return GreedySearch()
 
+    @staticmethod
+    def from_params_tflite(params):
```

Comment:
I think you can extend the base `from_params` method and pass an additional boolean argument `tflite_run` or `tflite_mode`.

Reply:
I removed the `from_params_tflite` function, and I extended the `from_params` function